tokens_definition_example_draft_v2.json

```diff
@@ -0,0 +1,205 @@
+{
+	"tokenCategories" : [
```

Comment:
Do we need the `token` namespace? I know it's a common practice in Liferay, but what do they add in the context of a `token_definition`? Can't we just make this `categories` and below, `sets`?

Reply:
👍  I am OK with leaving the token namespace out

Reply:
Maybe Brian won't like it... so we might as well just try.